jschon/catalog/__init__.py

```diff
@@ -42,7 +42,16 @@ def __call__(self, relative_path: str) -> JSONCompatible:
             filepath = str(filepath)
             filepath += self.suffix
 
-        return json_loadf(filepath)
+        try:
+            return json_loadf(filepath)
+        except OSError as e:
+            if e.filename is not None:
+                # The filename for OSError is not included in
+                # the exception args, which is what the Catalog
+                # puts in the CatalogError.  So it needs to be
+                # added separately for filesystem errors.
+                raise CatalogError(f'{e.strerror}: "{e.filename}"')
```

Comment:
Please just change `"{e.filename}"` to `{e.filename!r}`. This prints out the string using the standard library convention of single quotes (and also avoids escaping if the error message is JSON-ified).

Reply:
Good to know, will do!